src/index.ts

```diff
@@ -854,6 +870,64 @@ export abstract class PinejsClientCore<PinejsClient> {
 		}
 	}
 
+	private canRetryDefaultHandler(err: any) {
+		const code = err?.statusCode;
+		return !code || code === 429 || (code >= 500 && code < 600);
```

Comment:
I still have the question here of whether you are intentionally wanting to avoid `0` codes or whether it's an unintentional side effect since `!code` will also match `0`/`''`/`false`/etc as opposed to `code == null` which will only match `null`/`undefined`

Reply:
The intention is that no code or code 0 be retryable.  I'm not sure when a code 0 would actually occur?  But in any case, I would consider it to be a retryable case.

Reply:
Yeah I think I'd rather leave the `0` case as not retrying if we don't know why it would happen - we can always expand it later if we find that it does happen